<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceWidget</class>
 <widget class="QWidget" name="AppearanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appearance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabAppearance">
     <property name="currentIndex">
      <number>0</number>
     </property>

     <widget class="QWidget" name="AppPreviewTab">
      <attribute name="title">
       <string>PageEdit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="groupBoxWebView">
         <property name="title">
          <string>Fonts:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutPV">
          <item>
           <widget class="QLabel" name="labelWarning">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>If no font is specified in the CSS for your page, the following font will be used to display within PageEdit. These fonts will not be used in your actual ebook.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutPV" stretch="0,0">
            <item>
             <layout class="QGridLayout" name="gridLayoutPV">
              <item row="0" column="1">
               <widget class="QSpinBox" name="webViewFontSizeSpin">
                <property name="minimum">
                 <number>6</number>
                </property>
                <property name="maximum">
                 <number>99</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="labelWebViewFontSize">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Default font size to be used for PageEdit
if no font-size specified in your CSS</string>
                </property>
                <property name="text">
                 <string>Font Size:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="labelWebViewFontStandard">
                <property name="toolTip">
                 <string>Default font family to be used for PageEdit
if no font-family specified in your CSS</string>
                </property>
                <property name="text">
                 <string>Standard:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <spacer name="horizontalSpacerPV">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="3">
               <widget class="QFontComboBox" name="cbWebViewFontSerif"/>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="labelWebViewFontSerif">
                <property name="toolTip">
                 <string>Default font family to be used for PageEdit
if a serif font-family specified in your CSS</string>
                </property>
                <property name="text">
                 <string>Serif:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QFontComboBox" name="cbWebViewFontStandard"/>
              </item>
              <item row="4" column="3">
               <widget class="QFontComboBox" name="cbWebViewFontSansSerif"/>
              </item>
              <item row="4" column="2">
               <widget class="QLabel" name="labelWebViewFontSansSerif">
                <property name="toolTip">
                 <string>Default font family to be used for PageEdit
if a sans-serif font-family specified in your CSS</string>
                </property>
                <property name="text">
                 <string>Sans-Serif:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacerPV2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      
       <item>
        <widget class="QGroupBox" name="groupBox_PreviewPrint">
         <property name="title">
          <string>Printer Output Appearance</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="chkSkipPrintPreview">
            <property name="toolTip">
             <string>Skip Print Preview</string>
            </property>
            <property name="text">
             <string>Skip Print Preview</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lblPrintDPI">
              <property name="text">
               <string>Print DPI:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cboPrintDPI"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>

       <item>
        <widget class="QCheckBox" name="PreviewDarkInDM">
         <property name="text">
          <string>Make PageEdit simulate a dark appearance in dark mode</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>

       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="AppUiTab">
      <attribute name="title">
       <string>Main UI</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBoxIconSize">
         <property name="title">
          <string>Main Menu Icon Size:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelIconSize">
            <property name="text">
             <string>Adjust the size of the icons in the main menu.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="iconSizeSlider">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>18</number>
            </property>
            <property name="sliderPosition">
             <number>18</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpHighDPI">
         <property name="title">
          <string>High DPI Setting:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboHighDPI"/>
          </item>
          <item>
           <widget class="QLabel" name="lblHighDPIWarn">
            <property name="text">
             <string>(Needs PageEdit Restart)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>105</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="uifontGroup">
         <property name="title">
          <string>Fonts:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0">
          <item>
           <widget class="QLabel" name="uifontLabel">
            <property name="text">
             <string>UI Font:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editUIFont"/>
          </item>
          <item>
           <widget class="QPushButton" name="changeUIFontButton">
            <property name="toolTip">
             <string>Change the font used by PageEdit's menus, buttons, etc...</string>
            </property>
            <property name="text">
             <string>Change Font (Needs PageEdit Restart)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_51">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>

       <item>
        <widget class="QGroupBox" name="groupBoxSpecialCharacter">
         <property name="title">
          <string>Insert Special Characters:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayoutSC" stretch="0,0">
          <item>
           <layout class="QGridLayout" name="gridLayoutSC">
            <item row="0" column="0">
             <widget class="QLabel" name="labelSpecialCharacterFontSize">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Font size to be used for Insert Special Characters window</string>
              </property>
              <property name="text">
               <string>Font Size:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <spacer name="horizontalSpacerSC">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QSpinBox" name="specialCharacterFontSizeSpin">
              <property name="minimum">
               <number>6</number>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelSpecialCharacterFont">
              <property name="toolTip">
               <string>Font family to be used for Insert Special Characters window</string>
              </property>
              <property name="text">
               <string>Standard Font:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" colspan="2">
             <widget class="QFontComboBox" name="cbSpecialCharacterFont"/>
            </item>
            <item>
             <spacer name="horizontalSpacerSC2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacerTab">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>

    </widget>
   </item>
   <item>
    <spacer name="verticalSpacerReset">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutReset">
     <item>
      <widget class="QPushButton" name="resetAllButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Reset all fonts and colors to the default values</string>
       </property>
       <property name="text">
        <string>Reset All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
